substrate/frame/staking/src/pallet/mod.rs

```diff
@@ -72,7 +72,7 @@ pub mod pallet {
 	use crate::{BenchmarkingConfig, PagedExposureMetadata};
 
 	/// The in-code storage version.
-	const STORAGE_VERSION: StorageVersion = StorageVersion::new(17);
+	const STORAGE_VERSION: StorageVersion = StorageVersion::new(16);
```

Comment:
Already reset on chain to 16 https://polkadot.js.org/apps/?rpc=wss://westend-rpc.dwellir.com#/explorer/query/0xce9f22e996134d56a6dcd40572d2c36fa78cf07da44c3c2bdd232984ba2e0d37

Reply:
(For other reviewers)